test_data/issue-199.good.nix

```diff
@@ -14,15 +14,15 @@ let
       bla
     '' else ''
       blub
-    '')}
+     '')}
```

Comment:
this looks suspicious. like the multiline block ending doesn't get indented anymore.

Reply:
Yeah, this is just the consequence of "indenting the string block as a whole". This string is nested in the outer string, and, when we indent the outer string, we just keep the indentaiton of the inner one as is.